docs/PrestaShop.md

```diff
@@ -1,63 +1,93 @@
 
-## Support
+[![BTCPay Server Lundanode](https://img.youtube.com/vi/NjslXYvp8bk/mqdefault.jpg)](https://www.youtube.com/watch?v=NjslXYvp8bk "BTCPay Server - LunaNode")
 
-### Tested successfully
+For the self-hosted solutions, you'll have to wait for your node to sync fully before proceeding to step 3.
 
-* Prestashop 1.7.x
-* BTCPay server v1.0.1 and v1.0.2
+### 2.2 Third-party host
 
-### Contribute
+For those who want to test BTCPay out, or are okay with the limitations of a **third-party hosting** (dependency and privacy, as well as lack of some features) can use a one of many [third-party hosts](./ThirdPartyHosting.md).
 
-To contribute to this project, please fork and submit a pull request.
+The video below shows you how to connect your store to such a host.
 
-* [GitHub Issues](https://github.com/btcpayserver/prestashop-plugin/issues)
+[![BTCPay Server Third Party Host](https://img.youtube.com/vi/IT2K8It3S3o/mqdefault.jpg)](https://www.youtube.com/watch?v=IT2K8It3S3o "BTCPay Server - Third Party Host")
 
-### PrestaShop Support
+## 3. Pairing the store
 
-* [Homepage](http://www.prestashop.com)
-* [Documentation](http://doc.prestashop.com/)
-* [Support Forums](http://www.prestashop.com/forums/)
+BTCPay PrestaShop plugin is a **bridge between your server (payment processor) and your e-commerce store**.
+No matter if you're using a self-hosted or third-party solution from step 2, the pairing process is identical.
+
+1. In the `BTCPay server url`-field, enter the full URL of your host (including the https) – https://btcpay.mydomain.com
+2. Select the default transaction speed (this will change how much BTCPay recommends as transaction fee)
+3. Select the relevant order mode for your store (order is created before or after payment)
+4. Go to your BTCPay server, select a store, open its settings and select "Access Tokens".
+5. Click on "Create a new token", select your store and then approve.
+  5.1. You will see: "Server initiated pairing code: XXXX".
+6. Enter your pairing code.
+7. Press save. Prestashop will now attempt to make a connection with your BTCPayServer.
+8. A message will be shown if the connection was successful.
+
+## 4. Connecting your wallet
+
+No matter if you're using self-hosted or server hosted by a third-party, the process of configuring your wallet is the same.
+
+[![BTCPay Server Wallet](https://img.youtube.com/vi/xX6LyQej0NQ/mqdefault.jpg)](https://www.youtube.com/watch?v=xX6LyQej0NQ "BTCPay Server - Wallet")
+
+## 5. Testing the checkout
+
+Making a small test-purchase from your store will give you a piece of mind.
+Always make sure that everything is set up correctly before going live.
+
+The final video guides you through the steps of setting a gap limit in your Electrum wallet and testing the checkout process.
+
+[![BTCPay Server Checkout](https://img.youtube.com/vi/Fi3pYpzGmmo/mqdefault.jpg)](https://www.youtube.com/watch?v=Fi3pYpzGmmo "BTCPay Server - Wallet")
+
```

Comment:
All of this is copy-paste from Woo repo, and that Woo doc is also not that good.
Please remove this duplication (except store pairing?)

You can just give concise steps and reference to a adequate page.
1. Deploy BTCPay Server  https://docs.btcpayserver.org/Deployment/
2. Pair a store section
3. Connect a wallet https://docs.btcpayserver.org/WalletSetup/
4. Test the checkout is probably not needed.